### ISSUE TITLE
Vault PKI support?

### ISSUE THREAD

**Issue by azalio** [kelseyhightower/confd#731]
Hello!
Does confd support issuing certs and renewing procedure for them?

**Comment by okushchenko** [comment]
No, it doesn't. Also, I don't see how it might even be a responsibility of the confd tool to manage the certificates. How do you think this feature might be added here?

**Comment by azalio** [comment]
I'm going to use confd for ETCD and Vault values. And it might be convenient to use confd for vault PKI too because in another way I need to use one software for values from vault and another software for certs in vault. 
And from another point of view, certs just values for a common name (CN) as a key.

**Comment by okushchenko** [comment]
Do you want these value to be exposed along the key/value objects from Vault or as a completely separate backend?

**Comment by azalio** [comment]
I think a separate backend would be more useful than an integrated solution.

**Comment by okushchenko** [comment]
I'll leave this issue open for now. If anyone is interested in implementing such backend I'll gladly review the PR.

**Comment by tw3rp** [comment]
Created PR https://github.com/kelseyhightower/confd/pull/774 comments gladly welcome thanks!

**Comment by CH-rahulprabhu** [comment]
Small write up https://rprabhu.dev/2019/vault-confd-automation/

**Comment by tw3rp** [comment]
Small write up on how it works https://rprabhu.dev/2019/vault-confd-automation/
